src/admin-views/tribe-options-general.php

```diff
@@ -300,6 +300,16 @@
 
 $general_tab_fields += $tec_events_general_editing;
 
+// Our default tooltip.
+$trash_tooltip = esc_html__( 'This option allows you to automatically move past events to trash.', 'the-events-calendar' );
+// Some adjusted functionality with CT1 activated.
+if ( tribe()->getVar( 'ct1_fully_activated' ) ) {
+	$trash_tooltip = sprintf(
+		__( 'Trashed events will permanently be deleted in %d days, you can change that value using <code>EMPTY_TRASH_DAYS</code>.', 'the-events-calendar' ),
```

Comment:
Um, this really needs a note (`_x()`) so the constant doesn't get translated 😉 
OR - move it into a placeholder/value